app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,15 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str, favourite_dish: str,
+                 skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf blacksmith {self.nickname} with skill of "
+                + f"the {self._skill_level} level")
+
+    def get_rating(self) -> int:
+        return self._skill_level
```

Comment:
The `get_rating` method returns the `skill_level`, which is appropriate for this class. Ensure that this method aligns with the expected behavior in the context where it's used.